drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -741,30 +742,39 @@ def _get_zone_by_uid(self, uid):
     def _get_item_stats(self):
         """
         Returns a tuple representing the number of correctly-placed items,
-        and the total number of items that must be placed on the board (non-decoy items).
+        and the total number of items on the board (including decoy items).
         """
-        required_items, __, correct_items = self._get_item_raw_stats()
+        items = self._get_item_raw_stats()
 
-        return len(correct_items), len(required_items)
+        correct_count = len(items.correct_placed) + len(items.correct_decoy)
+        total_count = len(items.required) + len(items.decoy)
+
+        return correct_count, total_count
 
     def _get_item_raw_stats(self):
         """
-        Returns a 3-tuple containing required, placed and correct items.
+        Returns a named tuple containing required, decoy, placed, correct
+        placed, and correct decoy items.
```

Comment:
@arbrandes nit: `correct decoy` feels a bit weird - maybe `decoy items in bank`?